packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+
 version: 2.1.10
 
 environment:
-  sdk: ">=2.17.0 <4.0.0"
+  sdk: ">=2.19.0 <4.0.0"
```

Comment:
This would prevent Flutter 3.0 and 3.3 users from using this package, without any benefit.